src/qililab/experiment/base_experiment.py

```diff
@@ -21,6 +21,7 @@
 from threading import Thread
 
 import numpy as np
+import ruamel.yaml
```

Comment:
Instead of using this import and then `ruamel.yaml.YAML` we could import the YAML class directly:
```suggestion
from ruamel.yaml import YAML
```

Reply:
Indeed! It will look way better that way yes!

Reply:
Quickly applied this change!